src/app/views/sidebar/history/History.tsx

```diff
@@ -279,9 +312,10 @@ export class History extends Component<IHistoryProps, any> {
   }
 
 
-  private onDeleteQuery = (query: IHistoryItem) => {
+  private deleteQuery = async (query: IHistoryItem) => {
     const { actions } = this.props;
     if (actions) {
+      delete query.category;
```

Comment:
Since we are mutating the `query` object here, we should move the mutation to `removeHistoryItem` so that it's easier to find.